devopstools-jenkins-chefhabitat-terraform/azuredeploy.json

```diff
@@ -0,0 +1,391 @@
+{
+    "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "adminUsername": {
+            "type": "string",
+            "defaultValue": "",
```

Comment:
remove all of the defaultValues that are not valid...

Reply:
Removed